app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/MapsActivity.java

```diff
@@ -154,25 +287,30 @@ public void onMapReady(GoogleMap googleMap) {
     }
 
     private final void setUpMap() {
-        if (ActivityCompat.checkSelfPermission((Context) this, "android.permission.ACCESS_FINE_LOCATION") != 0) {
-            ActivityCompat.requestPermissions((Activity) this, new String[]{"android.permission.ACCESS_FINE_LOCATION"}, 1);
+        if (ActivityCompat.checkSelfPermission((Context) this,
```

Comment:
please add comments to this method

Reply:
... I'm starting to dislike auto-linter, if only for making things I didn't write look like my changes.